lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -953,6 +953,26 @@ export class BrowserCacheManager extends CacheManager {
         this.internalStorage.clear();
     }
 
+    /**
+     * Clears all access tokes that have claims prior to saving the current one
+     * @param credential 
+     * @returns 
+     */
+    async clearTokensAndKeysWithClaims(): Promise<void> {
+
+        const tokenKeys = this.getTokenKeys();
+            
+        const removedAccessTokens: Array<Promise<void>> = [];
+        tokenKeys.accessToken.forEach((key: string) => {
+            // if the access token has claims in its key, remove the token key and the token
+            const credential = this.getAccessTokenCredential(key);
+            if(credential?.requestedClaimsHash && key.includes(credential.requestedClaimsHash)) {
+                removedAccessTokens.push(this.removeAccessToken(key));
```

Comment:
I do see `Info` of "BrowserCacheManager: remoteTokenKey - accessToken removed from map` but am wondering if we can add a `Warning` here on config change as well. I think it is warranted as these should only occur once under very specific config changes, and will make it easier for us to track in telemetry

Reply:
Added warning if `tokens` are deleted.